ops/gulp/dev-watch.js

```diff
@@ -13,7 +13,10 @@ const task = () => {
 	gulp.watch([`./docs/**/*.html`,`./src/**/*.html`], ['docs-html']);
 
 	//watch component styles
-	gulp.watch(`./src/**/*.scss`, ['pkg-css']);
+	gulp.watch([`./src/**/*.scss`], ['pkg-css']);
+
+	//watch icons 
+	gulp.watch([`./src/icons/**/*.*`], ['pkg-icons']);
```

Comment:
Not much harm here, but I don't think we actually need to watch this. Adding new icons should be rare enough that we could explicitly run `gulp icons-build` but regardless this should kick off `icons-build` at which point a new `icons.scss` will get added to the `src/styles` directory and be picked up by the `pkg-css` task.